.github/workflows/benchmark.yaml

```diff
@@ -10,22 +10,10 @@ jobs:
     name: Benchmark
     runs-on: ubuntu-latest
     steps:
-      - name: Setup Go
-        uses: actions/setup-go@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
```

Comment:
I'm switching order here based on https://github.com/actions/setup-go - they seem to recommend first, then setup-go.

I guess it makes sense given we want `setup-go` to do all the caching too.